src/routes/auth/unlock/+page.svelte

```diff
@@ -38,13 +53,42 @@
         let ownIdentity = await MultipassStoreInstance.getOwnIdentity()
         ownIdentity.fold(
             (_: any) => {
+                localStorage.setItem('pin', pin.toString())
+                localStorage.setItem('is_user_logged', 'true')
                 goto(Route.NewAccount)
             },
             (_: any) => {
                 goto(Route.Pre)
             }
         )
     }
+
+    async function doAutoLoginIfUserExist() {
+            // HACK(Lucas): Temp Solution to avoid clearing cache all time
+            let username = localStorage.getItem('user_name')
+            let statusMessage = localStorage.getItem('status_message')
+            let pin = localStorage.getItem('pin')
```

Comment:
We shouldnt store the pin anyplace in plaintext as this opens up security concerns, even if temporary.